src/TabNav.tsx

```diff
@@ -45,7 +59,8 @@ type StyledTabNavLinkProps = {
 const TabNavLink = styled.a.attrs<StyledTabNavLinkProps>(props => ({
   activeClassName: typeof props.to === 'string' ? 'selected' : '',
   className: classnames(ITEM_CLASS, props.selected && SELECTED_CLASS, props.className),
-  role: 'tab'
+  role: 'tab',
+  'aria-selected': props.selected ? true : false
```

Comment:
```suggestion
  'aria-selected': !!props.selected
```

Reply:
@kendallgassner thinking about this - can't we just do:
```suggestion
  'aria-selected': props.selected
```

I think I was setting it to something else originally and wasn't really thinking it through 😅

Reply:
yes!